src/Keyboard.cpp

```diff
@@ -36,25 +36,25 @@ static const uint8_t _hidReportDescriptor[] PROGMEM = {
     0x85, 0x02,                    //   REPORT_ID (2)
     0x05, 0x07,                    //   USAGE_PAGE (Keyboard)
    
-  0x19, 0xe0,                    //   USAGE_MINIMUM (Keyboard LeftControl)
+    0x19, 0xe0,                    //   USAGE_MINIMUM (Keyboard LeftControl)
```

Comment:
My main goal with the patch was to avoid the spurious indentation that gave the false impression it was describing of the descriptor structure. The rest was "couldn't resist some glaring misalignment".

Reply:
What you point to as inconsistency here is actually deliberate indentation of the COLLECTION unit in the the descriptor.

Reply:
It was clarifying the descriptor structure that was the initial goal of my patch.

Reply:
I see now. Thanks for clarifying.